modules/move-etcd-different-disk.adoc

```diff
@@ -37,116 +77,7 @@ $ oc debug node/<node_name>
-          ExecStart=/usr/lib/systemd/systemd-makfs.xfs -f /dev/<new_disk_name>
-          TimeoutSec=0
-
-          [Install]
-          WantedBy=var-lib-containers.mount
-        enabled: true
-        name: systemd-mkfs@dev-<new_disk_name>.service
-      - contents: |
-          [Unit]
-          Description=Mount /dev/<new_disk_name> to /var/lib/etcd
-          Before=local-fs.target
-          Requires=systemd-mkfs@dev-<new_disk_name>.service
-          After=systemd-mkfs@dev-<new_disk_name>.service var.mount
-
-          [Mount]
-          What=/dev/<new_disk_name>
-          Where=/var/lib/etcd
-          Type=xfs
-          Options=defaults,prjquota
-
-          [Install]
-          WantedBy=local-fs.target
-        enabled: true
-        name: var-lib-etcd.mount
-      - contents: |
-          [Unit]
-          Description=Sync etcd data if new mount is empty
-          DefaultDependencies=no
-          After=var-lib-etcd.mount var.mount
-          Before=crio.service
-
-          [Service]
-          Type=oneshot
-          RemainAfterExit=yes
-          ExecCondition=/usr/bin/test ! -d /var/lib/etcd/member
-          ExecStart=semanage fcontext -a -e /sysroot/ostree/deploy/rhcos/var/lib/etcd/ /var/lib/etcd/
-          ExecStart=/bin/rsync -ar /sysroot/ostree/deploy/rhcos/var/lib/etcd/ /var/lib/etcd/
-          TimeoutSec=0
-
-          [Install]
-          WantedBy=multi-user.target graphical.target
-        enabled: true
-        name: sync-var-lib-etcd-to-etcd.service
-      - contents: |
-          [Unit]
-          Description=Restore recursive SELinux security contexts
-          DefaultDependencies=no
-          After=var-lib-etcd.mount
-          Before=crio.service
-
-          [Service]
-          Type=oneshot
-          RemainAfterExit=yes
-          ExecStart=/sbin/restorecon -R /var/lib/etcd/
-          TimeoutSec=0
-
-          [Install]
-          WantedBy=multi-user.target graphical.target
-        enabled: true
-        name: restorecon-var-lib-etcd.service
-----
-
-. Log in to the cluster as a user with `cluster-admin` privileges and create the machine configuration:
-+
-[source,terminal]
-----
-$ oc login -u <username> -p <password>
-----
-+
-[source,terminal]
-----
-$ oc create -f etcd-mc.yml
-----
-+
-The nodes are updated and rebooted. After the reboot completes, the following events occur:
-+
-* An XFS file system is created on the specified disk.
-* The disk mounts to `/var/lib/etcd`.
-* The content from `/sysroot/ostree/deploy/rhcos/var/lib/etcd` syncs to `/var/lib/etcd`.
-* A restore of `SELinux` labels is forced for `/var/lib/etcd`.
-* The old content is not removed.
-
-. After the nodes are on a separate disk, update the `etcd-mc.yml` file with contents such as the following, replacing instances of `<new_disk_name>` with the noted device name:
+. Create a `MachineConfig` YAML file named `etcd-mc.yml` as follows. This machine configuration file is for device names that begin with `/dev/sd*`.
```

Comment:
I still see the sentence I referred to above. I think this can just be the first sentence and maintain meaning. :slightly_smiling_face: 

```adoc
. Create a `MachineConfig` YAML file named `etcd-mc.yml` with contents such as the following:
```

I added a little bit to it to be consistent with how some other areas of the etcd docs are written.